src/dist/docbook-xsl/fo-pdf.xsl

```diff
@@ -102,14 +102,14 @@
   </xsl:param>
 
   <xsl:param name="title.font.family">
-    <xsl:call-template name="pickfont-serif"/>
+    <xsl:call-template name="pickfont-sans"/>
   </xsl:param>
 
   <!--
     Text properties
   -->
 
-  <xsl:param name="hyphenate">false</xsl:param>
+  <xsl:param name="hyphenate">true</xsl:param>
```

Comment:
Here's the pull request: https://github.com/asciidoctor/asciidoctor-fopub/pull/63